cmd/jujud/agent/machine/manifolds.go

```diff
@@ -411,14 +411,14 @@ func commonManifolds(config ManifoldsConfig) dependency.Manifolds {
 
 		// The multiwatcher manifold watches all the changes in the database
 		// through the AllWatcherBacking and manages notifying the multiwatchers.
-		multiwatcherName: ifDatabaseUpgradeComplete(ifController(multiwatcher.Manifold(multiwatcher.ManifoldConfig{
+		multiwatcherName: ifDatabaseUpgradeComplete(multiwatcher.Manifold(multiwatcher.ManifoldConfig{
```

Comment:
Why would these be run on non-controllers?

Reply:
```
  multiwatcher:
    error: '"upgrade-database-flag" not running: dependency not available'
    inputs:
    - state
    - upgrade-database-flag
    state: stopped
```

Reply:
maybe add comments to these transitive ones to avoid confusion

Reply:
Done.